packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -342,4 +409,48 @@ export default {
         display: block;
         font-weight: normal;
     }
+
+    .c-formField-icon {
+        svg {
+            position: absolute;
+            height: $form-input-iconSize;
+            width: $form-input-iconSize;
+
+            path {
+                fill: $color-content-subdued;
+            }
+        }
+    }
+
+    .c-formField-icon--disabled {
+        svg {
+            path {
+                fill: $color-content-disabled;
+            }
+        }
+    }
+
+    .c-formField-icon--small {
```

Comment:
I think you could potentially get rid of the extra classnames for these sizes, and use the formfield size class like this:

```scss
    .c-formField-icon {
        .c-formField-field--small ~ & {
            @include indent-icon('bottom', $form-input-icon-verticalIndent--small);
        }
        .c-formField-field--medium ~ & {
            @include indent-icon('bottom', $form-input-icon-verticalIndent);
        }
        …
    }
```

Reply:
This sadly doesn't work for the dropdown because it's no longer a sibling

Reply:
You should be able to change this for this component, as the styles are duplicated inside the drop down component separately?

Reply:
I _think_ that might just be for the caret in the dropdown `c-formDropdown-icon`? This is for the leading/trailing icon